### PR TITLE
update README

### PR DESCRIPTION
**Pull request by dingyishen-amazon**
### Issue #, if available:
Github issues: 
https://github.com/aws/amazon-chime-sdk-ios/issues/311
https://github.com/aws/amazon-chime-sdk-ios/issues/315

### Description of changes:
Update IOS set up README

Improve our README and set up instructions to include what builders need to update based on their own developer account.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,22 +49,31 @@ Download the `AmazonChimeSDK` and `AmazonChimeSDKMedia` binaries from the latest
 
 Unzip and copy the `.framework`s or `.xcframework`s to `root`.
 
+For Xcode12.3 and later, starting from Amazon Chime SDK iOS v0.15.0, download .xcframeworks instead of .frameworks under `root` folder.
```
Comment (ziyiz-amzn) [inline README.md:52]: Please refer to quip doc and update.

File: README.md [hunk 2]
```diff
@@ -49,22 +49,31 @@ Download the `AmazonChimeSDK` and `AmazonChimeSDKMedia` binaries from the latest
...
+
 ### 2. Update project file
 
-Open your `.xcodeproj` file in Xcode and click on your build target. Under `Build Settings` tab, add `$(PROJECT_DIR)` to `Framework Search Path`
+Open your `.xcodeproj` file in Xcode and click on your build target.
```
Comment (ziyiz-amzn) [inline README.md:56]: Use numbered bullet points for steps. valid through the entire PR.

File: README.md [hunk 3]
```diff
@@ -49,22 +49,31 @@ Download the `AmazonChimeSDK` and `AmazonChimeSDKMedia` binaries from the latest
...
+    * `-ObjC`
 
-After adding the two frameworks, verify that `Embed & Sign` is selected under the `Embed` option.
+<img src="./media/build_setting.png" alt="image" width="500"/>
 
-In `Build Settings` tab, under `Linking` section, add the following two flags in `Other Linker Flags`:
+Under `General` tab, look for `Frameworks, Libraries, and Embedded Content` section. Click on +, then Add Others, then Add Files.
+
+* If you are using traditional `.framework`, specify the location of `AmazonChimeSDK.framework` and `AmazonChimeSDKMedia.framework` from Step 1.
+* If you are using `.xcframework`, specify the location of `AmazonChimeSDK.xcframework` and `AmazonChimeSDKMedia.xcframework` from Step 1. If encounter compile error when using traditional `.framework`, which occurs in Xcode 12.3 and later, please use `.xcframework` instead.
```
Comment (ziyiz-amzn) [inline README.md:72]: I updated this section in the quip doc, please update accordingly.

**Review by ziyiz-amzn — APPROVED** [review]

**Review by hokyungh — APPROVED** [review]
LGTM. Nice pics.

**Pull request merged**